api/src/devcomp/infrastructure/datasources/learning-content/samples/elements/qcu-discovery.sample.js

```diff
@@ -0,0 +1,14 @@
+import { randomUUID } from 'node:crypto';
+
+export function getQCUDiscoverySample(nbOfProposals = 4) {
+  return {
+    id: randomUUID(),
+    type: 'qcu-discovery',
+    instruction: '<p>Une question découverte ?</p>',
+    proposals: Array.from(Array(nbOfProposals)).map((_, i) => ({
+      id: `${i + 1}`,
+      content: `Proposition ${i + 1}`,
+      feedback: { diagnosis: `<p> Constat ${i + 1}</p>` },
```

Comment:
on s'en fiche un peu mais 
```suggestion
      feedback: { diagnosis: `<p> Diagnostique ${i + 1}</p>` },
```